templates/CRM/common/jsortable.tpl

```diff
@@ -151,7 +151,7 @@
 
   //plugin to sort on currency
   cj.fn.dataTableExt.oSort['currency-asc']  = function(a,b) {
-    var symbol = "{/literal}{$defaultCurrencySymbol}{literal}";
+    var symbol = "{/literal}{if isset($defaultCurrencySymbol)}{$defaultCurrencySymbol}{/if}{literal}";
```

Comment:
(This bit appears the same in both variants.) So there's a warning about this being undefined while running `CRM_Core_FormTest`?

The variable here is exported by way of `CRM_Core_Page::run()`, which would be part of most page-views... but I guess it's not part of `CRM_Core_FormTest`. For the test-scenario, it's sorta spurious because this generic unit-test probably doesn't really need `defaultCurrencySymbol`. So I can see why one might just wanna hide it this way.

But... if `$defaultCurrencySymbol` was ever blank at runtime, then the warning would be appropriate. (Incidentally, from skimming `Page::run()`, it looks like this error might happen if the page-request uses snippet/print-mode?)

I'd be tempted to make the `assign('defaultCurrencySymbol'...)` more reliable. Like... What if it moved from

https://github.com/civicrm/civicrm-core/blob/9de5aa1b82d4db922a01f834de5488e204cad0f5/CRM/Core/Page.php#L203

to be alongside

https://github.com/civicrm/civicrm-core/blob/01a5461df0f17518505f45d3f59eaa6e895083ea/CRM/Core/Smarty.php#L142-L144

?